sopel/irc.py

```diff
@@ -202,13 +186,14 @@ def initiate_connect(self, host, port):
             self.send = self._ssl_send
             self.recv = self._ssl_recv
         elif not has_ssl and self.config.core.use_ssl:
-            stderr('SSL is not avilable on your system, attempting connection '
-                   'without it')
+            LOGGER.warning(
+                'SSL is not avilable on your system, '
```

Comment:
Let's fix this ancient grammatical mess as long as it's already getting touched.

```suggestion
                'SSL is not available on your system; '
```